src/components/Form/Amount/index.tsx

```diff
@@ -45,7 +46,7 @@ const Amount = ({ className, register, max, setValue, assetSuffix, error }: Amou
             </button>
             <button
               className="text-accent-content underline hover:opacity-70 mx-1 font-semibold"
-              onClick={() => setValue(roundNumber(Number(max)))}
+              onClick={() => setValue(roundNumber(Number(max) - BALANCE_LEFT_FOR_TX_APPROVAL))}
```

Comment:
A user that has `0.00` tokens available will now enter `-0.5` when clicking on the 'MAX' button. Let's maybe wrap this with
```suggestion
              onClick={() => setValue(Math.max(roundNumber(Number(max) - BALANCE_LEFT_FOR_TX_APPROVAL, 0))}
```

Reply:
@ebma updated ✅